src/dom/dom.js

```diff
@@ -19,13 +19,13 @@
 import p5 from '../core/main';
 
 /**
- * Searches the page for the first element that matches the given CSS selector string (can be an
- * ID, class, tag name or a combination) and returns it as a <a href="#/p5.Element">p5.Element</a>.
- * The DOM node itself can be accessed with .elt.
- * Returns null if none found. You can also specify a container to search within.
+ * Searches the page for the first element that matches the given CSS selector string (can be an `#id`, a `.class`,
+ * a tag name, or a combination) and returns it as a <a href="#/p5.Element">p5.Element</a>.
+ * The DOM node itself can be accessed with `.elt`.
+ * Returns `null` if none found. You can also specify a `container` to search within.
```

Comment:
I'm not sure "container" here needs to be enclosed in code tags as it's not referencing a very specific item in this case.

Reply:
Same for below.